zone/trading.cpp

```diff
@@ -774,20 +775,22 @@ void Client::FinishTrade(Mob* tradingWith, bool finalizer, void* event_entry, st
 				}
 
 				const EQ::ItemData* item = inst->GetItem();
-
-				bool isPetAndCanHaveNoDrop = (RuleB(Pets, CanTakeNoDrop) &&
-					_CLIENTPET(tradingWith) &&
-					tradingWith->GetPetType()<=petOther);
+				const bool isPet = _CLIENTPET(tradingWith) && tradingWith->GetPetType()<=petOther;
+				const bool PetsCanTakeQuestItems = RuleB(Pets, CanTakeQuestItems);
+				const bool isPetAndCanHaveNoDrop = (RuleB(Pets, CanTakeNoDrop) &&	isPet);
+				const bool isPetAndCanHaveQuestItems = (PetsCanTakeQuestItems &&	isPet);
```

Comment:
Snake-case is preferred. `is_pet`, `can_take_quest_items`, `can_have_no_drop`, and `can_have_quest_item`.

Reply:
I guess i just default to trying to be consistent with the code that i'm working in. But the preference is to change to change to snake-case regardless of consistency in the code area your in? Not a big deal either way, just want to be sure.